docs/scenarios/nfs-pv/README.md

```diff
@@ -0,0 +1,7 @@
+## Migrating a **stateful** OpenShift app with **NFS PVs**
+
+This scenario walks through Migration of a stateful OpenShift app whose Persistent Volume (PV) claims tie into [NFS (Network File System)](https://en.wikipedia.org/wiki/Network_File_System) storage.
+
+
```

Comment:
For NFS we support 2 actions:

Copy, a new PV will be used on the destination cluster and data from the source PV will be copied to the destination PV.

Move, the existing PV on the source cluster will be detached from source and attached to destination.  We call this 'Swinging the PV'.

Reply:
Added